index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+   body {
+     background: rgb(255, 236, 29);
+   }
+   .picture {
+     size: 300px;
```

Comment:
`size` isn't a valid CSS property. What's your intention here?

(the validator will catch this)

Reply:
I meant to re-size the image using CSS. Upon searching, it seems like I can only do this with HTML. Something like: 

```suggestion
     size: 300px;
    
     <img class="picture" src="https://cdn.mos.cms.futurecdn.net/42E9as7NaTaAi4A6JcuFwG-1024-80.jpg" alt="bananas">
```

Reply:
I meant to re-size the image using CSS. Upon searching, it seems like I can only do this with HTML. Something like:

```<img class="picture" src="https://cdn.mos.cms.futurecdn.net/42E9as7NaTaAi4A6JcuFwG-1024-80.jpg" alt="bananas" width="200" height="200"
```

Reply:
I meant to re-size the image using CSS. Upon searching, it seems like I can only do this with HTML. Something like:
```suggestion
     <img class="picture" src="https://cdn.mos.cms.futurecdn.net/42E9as7NaTaAi4A6JcuFwG-1024-80.jpg" alt="bananas" height="200" width="200">
```